python/google/protobuf/pyext/descriptor.cc

```diff
@@ -58,6 +58,43 @@
               : 0)                                               \
        : PyBytes_AsStringAndSize(ob, (charpp), (sizep)))
 
+#if PY_VERSION_HEX < 0x030900B1 && !defined(PYPY_VERSION)
+static inline PyCodeObject* PyFrame_GetCode(PyFrameObject *frame)
```

Comment:
We do not generally use `static inline`, just `static` should be sufficient. The compiler will almost certainly inline a function as small as this.